fighthealthinsurance/common_view_logic.py

```
@@ -1646,3 +1651,124 @@ async def format_response(response: dict[str, str]) -> str:
+        except Exception as e:
+            logger.opt(exception=True).error("Error processing checkout session")
+            raise e
+
+    @staticmethod
+    def handle_checkout_session_expired(request, session):
+        try:
+            # TODO: More complete handling
+            logger.debug(f"Checkout session expired: {session}")
+            try:
+                metadata = session.metadata
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)